src/nacl/public.py

```diff
@@ -81,6 +89,14 @@ def __init__(self, private_key, encoder=encoding.RawEncoder):
     def __bytes__(self):
         return self._private_key
 
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
+        return bytes(self) == bytes(other)
```

Comment:
Thanks for bearing with us on this btw, I apologize for the long, long (long) wait.